simulations/cdee1a97df0e954187f50a5615a4ee3c_equipment.csv

```diff
@@ -0,0 +1,2 @@
+EquipmentID,EquipmentName
```

Comment:
Please delete this, and all other *.csv test results

Reply:
Done in the new commit